lib/apiservers/engine/backends/container.go

```diff
@@ -217,38 +217,103 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
+	// these are the parameters that the containers were actually created with
+	layer.Config.Cmd = config.Config.Cmd
+	layer.Config.WorkingDir = config.Config.WorkingDir
+	layer.Config.Entrypoint = config.Config.Entrypoint
+	layer.Config.Env = config.Config.Env
+	layer.Config.AttachStdin = config.Config.AttachStdin
+	layer.Config.AttachStdout = config.Config.AttachStdout
+	layer.Config.AttachStderr = config.Config.AttachStderr
+	layer.Config.Tty = config.Config.Tty
+	layer.Config.OpenStdin = config.Config.OpenStdin
+	layer.Config.StdinOnce = config.Config.StdinOnce
+	layer.ContainerID = createResults.Payload.ID
+
+	viccontainer.GetCache().SaveContainer(createResults.Payload.ID, layer)
+
 	// Success!
 	log.Printf("container.ContainerCreate succeeded.  Returning container handle %s", *createResults.Payload)
 	return types.ContainerCreateResponse{ID: id}, nil
 }
 
+// ContainerKill sends signal to the container
+// If no signal is given (sig 0), then Kill with SIGKILL and wait
+// for the container to exit.
+// If a signal is given, then just send it to the container and return.
 func (c *Container) ContainerKill(name string, sig uint64) error {
 	return fmt.Errorf("%s does not implement container.ContainerKill", c.ProductName)
 }
 
+// ContainerPause pauses a container
 func (c *Container) ContainerPause(name string) error {
 	return fmt.Errorf("%s does not implement container.ContainerPause", c.ProductName)
 }
 
+// ContainerRename changes the name of a container, using the oldName
+// to find the container. An error is returned if newName is already
+// reserved.
 func (c *Container) ContainerRename(oldName, newName string) error {
 	return fmt.Errorf("%s does not implement container.ContainerRename", c.ProductName)
 }
 
+// ContainerResize changes the size of the TTY of the process running
+// in the container with the given name to the given height and width.
 func (c *Container) ContainerResize(name string, height, width int) error {
-	return fmt.Errorf("%s does not implement container.ContainerResize", c.ProductName)
+	defer trace.End(trace.Begin("ContainerResize"))
+
+	// Get an API client to the portlayer
+	client := PortLayerClient()
+	if client == nil {
+		return derr.NewErrorWithStatusCode(fmt.Errorf("container.ContainerResize failed to create a portlayer client"),
+			http.StatusInternalServerError)
+	}
+
+	// Call the port layer to resize
+	plHeight := int32(height)
+	plWidth := int32(width)
+	plResizeParam := interaction.NewContainerResizeParams().WithID(name).WithHeight(plHeight).WithWidth(plWidth)
+
+	_, err := client.Interaction.ContainerResize(plResizeParam)
+	if err != nil {
+		if _, isa := err.(*interaction.ContainerResizeNotFound); isa {
+			return derr.NewRequestNotFoundError(fmt.Errorf("No such container: %s", name))
+		}
+
+		// If we get here, most likely something went wrong with the port layer API server
+		return derr.NewErrorWithStatusCode(fmt.Errorf("Unknown error from the interaction port layer"),
```

Comment:
Can you log the error and append it's string to this `fmt` string?  `fmt.Errorf("Unknown error {....}: %s", err)`.